.github/workflows/ci.yml

```diff
@@ -11,7 +11,8 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
```

Comment:
should run on ubuntu 20 instead of removing it completely?

Reply:
Refer this - https://github.com/loopbackio/loopback-connector-mysql/pull/476#discussion_r1056980407 @dhmlau . There is a separate issue to track that. It will need a lot of changes in test cases first as currently it completely fails on mysql8. and ubuntu20 doesn't have support for mysql5.7

Reply:
Here is the issue as well.